pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -156,20 +189,21 @@ func TestRESTDelete(t *testing.T) {
 }
 
 func TestRESTList(t *testing.T) {
+	fakeClock := clocktesting.NewFakeClock(time.Now())
```

Comment:
nit: introduce `now := time.Now()` for consistency with other tests and less verbosity